dev/AppLifecycle/ExecuteCommandBase.h

```diff
@@ -80,13 +80,13 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
         }
 
     protected:
-        winrt::com_ptr<IUnknown> m_site;
-        winrt::com_ptr<IShellItemArray> m_selection;
-        std::wstring m_directory;
-        DWORD m_keyState;
-        BOOL m_noShowUI;
-        std::wstring m_parameters;
-        POINT m_pos;
-        int m_showWindowState;
+        winrt::com_ptr<IUnknown> m_site{ nullptr };
```

Comment:
You only need to init the value types.  No need to initialize com_ptr or wstring.

Reply:
Got it!

Reply:
Moved to and addressed in https://github.com/microsoft/WindowsAppSDK/pull/4363